usecases/Concrete/knowledgeGraph/emodul/validation.py

```diff
@@ -0,0 +1,78 @@
+from pyshacl import validate
```

Comment:
not sure that that file does, but all files should eiter be in lebedigital (the actual functions), in tests (the tests for these functions) or in minimumWorkingExample (the pydoit workflow that processes the data for the minimum working example). The folder usecases/Concrete is deprecated.

Reply:
That file is not used anymore, I think that was an accidental commit

Reply:
This file is still in Concrete, please remove it?